hack/lib/vars.bash

```diff
@@ -59,7 +59,7 @@ export OLM_UPGRADE_CHANNEL="${OLM_UPGRADE_CHANNEL:-"$OLM_CHANNEL"}"
 export OLM_SOURCE="${OLM_SOURCE:-"$OPERATOR"}"
 export TEST_KNATIVE_UPGRADE="${TEST_KNATIVE_UPGRADE:-true}"
 export TEST_KNATIVE_E2E="${TEST_KNATIVE_E2E:-true}"
-export TEST_KNATIVE_KAFKA="${TEST_KNATIVE_KAFKA:-false}"
+export TEST_KNATIVE_KAFKA="${TEST_KNATIVE_KAFKA:-true}"
```

Comment:
This must not change I think, or we'll render all the `with-kafka` stuff that we did void.

Reply:
@markusthoemmes could you please check with @matzew -- last I hear this was what we wanted to do? whichever works -- I just want to be sure

Reply:
CI now always runs the `test-e2e-with-kafka` target, so we keep all these flag defaulted to false and only set them to true on that branch.